doc/design/OrionHDFSSink.md

```diff
@@ -225,13 +232,13 @@ Use `OrionHDFSSink` if you are looking for a JSON or CSV-based document storage
 ###<a name="section4.1"></a>`OrionHDFSSink` class
 As any other NGSI-like sink, `OrionHDFSSink` extends the base `OrionSink`. The methods that are extended are:
```

Comment:
It should be avoided to include too much deep developer-oriented information in users manuals, e.g. this part showing method signature is a big "hard". I'd suggest to analyze this globally and do the proper documentation refator if needed.